README.md

```diff
@@ -114,7 +147,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+We create ERDs to keep our data separate from our logic code. Ultimately we use
+the ERD file to visualize and describe our data away from the file that will
+manipulate the data.
 ```
```

Comment:
This is more or less true, but perhaps most importantly, ERDs describe relationships between entities, like pets, owners, vets for examples.